app/Http/Middleware/Theme.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Session;
+
+class Theme
+{
+    public function handle($request, Closure $next): mixed
+    {
+        // Set theme if exists
+        if (Session::has('backpack.ui.view_namespace')) {
+            Config::set('backpack.ui.view_namespace', Session::get('backpack.ui.view_namespace'));
+        }
```

Comment:
```suggestion
Config::set('backpack.ui.view_namespace', Session::get('backpack.ui.view_namespace') ?? config('backpack.ui.view_namespace'));
```

Reply:
@pxpm why do we need to set config value using config value when no session was set? Wouldn't that be the same as doing `setConfig(key) = config(key)` or `1 = 1` 🤔 

But maybe I am missing a case in which this is needed 😄

Reply:
Fair point!